docs/reporters.md

```diff
@@ -0,0 +1,100 @@
+---
+title: InSpec Reporters
+---
+
+# InSpec Reporters
+
+InSpec allows you to output your test results to one or more reporters. You can configure the reporter(s) using either the `--json-config` option or the `--reporter` option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
+
+## Syntax
+You can specify one or more reporters using the `--reporter` cli flag. You can also specify a output by appending a path seperated by a colon.
+
+Output json to screen.
+
+```bash
+inspec exec --reporter json
+or
+inspec exec --reporter json:-
+```
+
+Output cli to screen, write json to a file.
+
+```bash
+inspec exec --reporter cli json:/tmp/output.json
+```
+Output nothing to screen, write junit and html to a file.
+
+```bash
+inspec exec --reporter junit:/tmp/junit.xml html:www/index.html
+```
+Output json to screen and redirect to file, write junit and html to a file.
```

Comment:
This is a bit of a weird example because we supply the functionality of writing to a file natively. Maybe consider piping to `tee -a` instead of redirecting to show how you can write to a file and display to the screen simultaneously?